ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
```

Comment:
Ah, I see. OK, I guess that evens out the probabilities: chances that all four output digits will get rounded to zero are always 1e-4, no matter what we have before the decimal point, as long as we have an integer-delimited uniform range. So I still maintain that we should either have it in all cases or none, for the sake of consistency. Probably cleanest to always include it in all cases.